res/values-in/strings.xml

```diff
@@ -103,9 +103,9 @@
     <string name="You">"Anda"</string>
     <string name="This_player_is_a_bot">"Pemain ini adalah bot."</string>
     <string name="This_player_is_not_signed_in">"Pemain ini tidak di-login."</string>
-    <string name="Online">"Online"</string>
+    <string name="Online">"Dalam Jaringan"</string>
     <string name="Unknown">"Tidak dikenal"</string>
-    <string name="Offline">"Luring"</string>
+    <string name="Offline">"Luar Jaringan"</string>
```

Comment:
"Luring" was already correct, according to other native speakers. I can see they are basically an abbreviation.